app/src/main/java/com/example/ihonboarding/MainActivity.kt

```diff
@@ -8,7 +8,7 @@ import androidx.compose.material.Surface
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.tooling.preview.Preview
-import com.example.ihonboarding.ui.theme.IHOnboardingTheme
+import com.example.ihonboarding.fdsa.theme.IHOnboardingTheme
```

Comment:
what is fdsa?

Reply:
oyah, forgot. I've deleted it in the new pull req